src/js/jsx/search/SearchBar.jsx

```diff
@@ -142,6 +170,29 @@ define(function (require, exports, module) {
         },
 
         /**
+         * Get the layer type and if it is linked or an artboard, as an array of strings
```

Comment:
Can you elaborate here what you're returning? For example, I think these are user-visible strings that describe something, but I'm not sure how they might be used.